src/main/java/io/vertx/kafka/admin/AdminUtilsWrapper.java

```diff
@@ -0,0 +1,116 @@
+package io.vertx.kafka.admin;
+
+import java.util.Map;
+import java.util.Properties;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.kafka.admin.impl.AdminUtilsWrapperImpl;
+import kafka.admin.RackAwareMode;
+import kafka.utils.ZkUtils;
+
+/**
+ * Provides a wrapper around important methods in Kafka's AdminUtils, namely
+ * @see kafka.admin.AdminUtils#createTopic(ZkUtils, String, int, int, Properties, RackAwareMode),
+ * @see kafka.admin.AdminUtils#deleteTopic(ZkUtils, String),
+ * @see kafka.admin.AdminUtils#changeTopicConfig(ZkUtils, String, Properties), and
+ * @see kafka.admin.AdminUtils#topicExists(ZkUtils, String)
+ */
+@VertxGen
+public interface AdminUtilsWrapper {
```

Comment:
Thanks for the quick reply. The name is not very-well chosen, you are right ;)
What about simply dropping the Wrapper at all? AdminUtils? Name clash should be no problem, because one usually doesn't use the vert.x version and the original class in the same class -- and if you really need that, you could prefix the Apache version with the full package name.